### DISCUSSION TITLE
font-medium Class Not Working on Firefox 123

### DISCUSSION THREAD

**Discussion by reasecret** [tailwindlabs/tailwindcss#13094] [Help]
I'm using Tailwind CSS v3.4.1 with Flowbite UI and Laravel. And I noticed that font-medium class is not working on Firefox. On Chrome there is no problem.

```
<span class="bg-blue-100 text-blue-800 text-xs font-medium me-2 px-2.5 py-0.5 rounded dark:bg-blue-900 dark:text-blue-300">Default</span>
<span class="bg-green-100 text-green-800 text-xs font-medium me-2 px-2.5 py-0.5 rounded dark:bg-green-900 dark:text-green-300">Green</span>
<span class="bg-red-100 text-red-800 text-xs font-medium me-2 px-2.5 py-0.5 rounded dark:bg-red-900 dark:text-red-300">Red</span>
```

Example:
Firefox
![resim](https://github.com/tailwindlabs/tailwindcss/assets/32717963/dc1dd457-606c-4125-bf15-6c6b3ec2fc56)

Chrome:
![resim](https://github.com/tailwindlabs/tailwindcss/assets/32717963/4540190f-24cd-4c5e-8b3e-86f081ec4471)

I've tried font-medium class on Tailwind playground:
https://play.tailwindcss.com/aFy1UZANjy

Working on Chrome but not working on Firefox. I didn't try other classes yet...

I don't know is this a browser problem or something else.

**Comment by RobinMalfait** [comment]
The `font-medium` class just sets a `font-weight: 500` in your CSS.

If the font you are loading doesn't support that font weight then it's not going to show up. Also double check that the font is actually loaded in Firefox.
